pkgs/tools/package-management/poetry/default.nix

```diff
@@ -39,10 +39,6 @@ let
           rev = version;
           hash = "sha256-Gc22Y2T4uO39jiOqEUFeOfnVCbknuDjmzFPZgk2eY74=";
         };
-        patches = [ ];
-        nativeCheckInputs = old.nativeCheckInputs ++ [
-          self.tomli-w
-        ];
```

Comment:
Can this override go away completely?

Reply:
Yes. This was only needed because poetry used a different version of poetry-core than what's in pythonPackages.